scenes/templates/scenes/playlists/indexoptions.html

```diff
@@ -0,0 +1,4 @@
+<option>---------</option>
+{% for playlist in playlists %}
+    <option>{{ playlist.name }}</option>
```

Comment:
As found out in #47 , we need a value for each option:
```suggestion
    <option value="{{ playlist.id }}">{{ playlist.name }}</option>
```